src/fps/AddressRegistry.sol

```diff
@@ -3,8 +3,10 @@ pragma solidity 0.8.15;
 import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 
 import {Test} from "forge-std/Test.sol";
+
 import {IAddressRegistry} from "src/fps/IAddressRegistry.sol";
 import {SUPERCHAIN_REGISTRY_PATH} from "src/fps/utils/Constants.sol";
+import {ETHEREUM_CHAIN_ID, SEPOLIA_CHAIN_ID} from "src/fps/utils/Constants.sol";
```

Comment:
Now that I think of it, we probably can reuse forge-std's `StdChains` to avoid having to maintain our own list of constants. And if there are chains forge-std is missing that we need, we can PR those into forge-std. Looks like forge-std doesn't have a way to just fetch chain ID from a chain alias but that's an easy helper method to upstream as well

Reply:
I would say this change is low priority and we shouldn't block on it. So I created https://github.com/ethereum-optimism/superchain-ops/issues/469 to track

@blmalone Can you update the tracking issue with this?

Reply:
I've added to the parent tracking issue: https://github.com/ethereum-optimism/superchain-ops/issues/396